docs/how-to-guides/working-with-images.md

```diff
@@ -105,6 +125,26 @@ come to expect.
 ```console
 docker build -t TAG .
 ```
+
+### Building Local Images
+
+To run your locally-built container images in Kubernetes manifests with `docker`:
+
+```
+# docker --namespace k8s.io build -t foo /some-dockerfile-directory
+# kubectl apply -f - <<EOF
```

Comment:
```suggestion
$ docker build -t foo /some-dockerfile-directory
$ kubectl apply -f - <<EOF
```

Change to `$` as above, plus `docker` doesn't use namespaces and has no `--namespace` option.